annotation/full_trigger.go

```diff
@@ -46,6 +49,12 @@ func (t *FullTrigger) Controlled() bool {
 	return t.Controller != nil
 }
 
+// Duplicated returns true if the full trigger is created from duplicating another full trigger;
+// otherwise returns false.
+func (t *FullTrigger) Duplicated() bool {
+	return t.CreatedFromDuplication
+}
+
```

Comment:
I think we can just use the exported field instead of having this method.

Encapsulation is good, but here I don't think it's really necessary :)

Reply:
Good point, removed.